scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -804,6 +804,7 @@ protected Object clone() {
 
 		private DtlsConnectorConfig config;
 		private boolean clientOnly;
+		private boolean recommendedCipherSuitesOnly = true;
```

Comment:
Why those 2 attributes (clientOnly, recommendedCipherSuitesOnly) are builder attributes ? 

I suspect this could be an issue for use case with  builder.getIncompleteConfig(); as we can not know if this boolean was modified or not. 
(E.g : [LeshanServerBuilder](https://github.com/eclipse/leshan/blob/leshan-1.0.0-M11/leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/LeshanServerBuilder.java#L380)

Reply:
> Just to let you know for leshan we probably don't need it. so it's just hypothetical.

Then we will see, if someone else is using the "incompleteConfig" :-).
(If you're interested, even if leshan don't need it, feel free to correct it.)

Reply:
It sounds good. :)

Reply:
Ok, so merge now?

Reply:
Yep !